app/components/spotlight/progress_bar_component.html.erb

```diff
@@ -0,0 +1,30 @@
+<div data-behavior='progress-panel' data-exhibit-id="<%= exhibit.id %>">
+  <div class="card index-status border-info mb-3">
+    <h3 class="card-header bg-info text-white h4"><%= t("#{translation_field}.heading") %></h3>
+    <div class="card-body">
+      <p data-behavior='monitor-start'>
+        <span class="text-muted" data-behavior='date'></span> <%= t("#{translation_field}.begin_html") %>
+      </p>
+
+      <% if I18n.exists?("#{translation_field}.current_html") %>
+        <p data-behavior='monitor-current'>
+          <span class="text-muted" data-behavior='date'></span> <%= t("#{translation_field}.current_html") %>
+        </p>
+      <% end %>
+
+      <% if I18n.exists?("#{translation_field}.completed_html") %>
+        <p data-behavior='monitor-completed'>
+          <span class="text-muted" data-behavior='date'></span> <%= t("#{translation_field}.completed_html") %>
+        </p>
+      <% end %>
+
+      <p class="bg-warning" data-behavior='monitor-error' style='display:none;'>
+        <%= t("#{translation_field}.error") %>
+      </p>
+
+      <div class="progress">
```

Comment:
Could use use the `<progress>` element? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

Reply:
No. This is a bootstrap class: https://getbootstrap.com/docs/4.0/components/progress/. Changing this to <progress class="progress"> it breaks the CSS.

Reply:
Isn't the semantic element better for accessibility than a bootstrap component?

Reply:
Probably. I think if we are trying to solve the accessibility issue then we should open a new ticket. This is the progress bar that was used in app/views/spotlight/catalog/_reindex_progress_panel.html.erb and app/views/spotlight/bulk_updates/_progress_panel.html.erb